app/screens/settings/general/settings.js

```diff
@@ -78,7 +78,7 @@ class Settings extends PureComponent {
         const {intl, navigator, theme} = this.props;
         navigator.push({
             screen: 'About',
-            title: intl.formatMessage({id: 'about.title', defaultMessage: 'About Mattermost'}),
+            title: intl.formatMessage({id: 'about.title', defaultMessage: 'About {appTitle}'}, {appTitle: this.props.config.SiteName || 'Mattermost'}),
```

Comment:
please add `config` to the above deconstruction and use that here and below

Reply:
👍